imports/core/ui/components/AppHeader.vue

```diff
@@ -1,61 +1,125 @@
 <template>
-  <header>
-    <div class="container">
+  <div :class="hidden ? 'hidden' : 'fixed'">
+    <header>
       <img class="logo" src="/images/logo_white.png" alt="logo" />
       <div class="stripe"></div>
       <h3>Voting</h3>
       <button @click="toggleSubmitProposalModal">Submit proposal</button>
-    </div>
-  </header>
+    </header>
+    <nav class="proposals-filter">
+      <button
+        v-for="f in filters"
+        :key="f"
+        :class="{active: currentFilter === f}"
+        @click="setFilter(f)"
+      >
+        {{f}}
+      </button>
+    </nav>
+  </div>
 </template>
 
 <script>
-  import { mapMutations } from 'vuex';
+  import { mapState, mapMutations } from 'vuex';
+
+  import { Proposals } from '/imports/core';
 
   export default {
+    props: {
+      hidden: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    data() {
+      return {
+        filters: [Proposals.filterTypes.NEWEST, Proposals.filterTypes.POPULAR],
+      };
+    },
     methods: mapMutations({
       toggleSubmitProposalModal: 'core/toggleSubmitProposalModal',
+      setFilter: 'voting/setFilter',
+    }),
+    computed: mapState({
+      currentFilter: store => store.voting.filter,
     }),
   }
 </script>
 
 <style lang="scss" scoped>
   @import "/imports/core/ui/styles/variables";
 
-  header {
-    position: relative;
-    flex-grow: 0;
-    flex-shrink: 0;
-    background: $brand-color;
-    @media screen and (max-width: $container-width){
-      padding: 10px;
+  .hidden, .fixed {
```

Comment:
![image](https://user-images.githubusercontent.com/3524016/30800725-2689fd66-a20b-11e7-8e0a-c7dd6b30fe58.png)
The header is now covering the first proposal item in list.

Reply:
I can't reproduce (Chromium 61, Ubuntu).
![2017-09-26 18-14-07](https://user-images.githubusercontent.com/9007851/30849692-85a5191c-a2e6-11e7-893a-68abc4161b53.png)